tests/test_home.py

```diff
@@ -0,0 +1,36 @@
+from bank_api.resources.login_resource import LOGIN_ENDPOINT
+from bank_api.resources.users_resource import USERS_ENDPOINT
+from bank_api.resources.home_resource import HOME_ENDPOINT
+
+def test_home(client):
+    # Create a example of user
+    user_json = {
+            "first_name": "jhon",
+            "last_name": "doe",
+            "type": "user",
+            "document_id": "12345678",
+            "birthday": "1997-01-01",
+            "country": "peru",
+            "city": "lima",
+            "address": "av siempreviva",
+            "email": "ca@texample.com",
+            "phone_number": "999555999"
+    } 
+    # User data to login
+    user_creation_response = client.post(f"{USERS_ENDPOINT}",json=user_json)
+    print(user_creation_response.json)
+    login_json = {
+        "username": user_creation_response.json["username"],
+        "password": user_creation_response.json["password"],
+        "code": user_creation_response.json["code"] # Esta pasando algo con el tipo de dato de code, me lo lee cuando es un string y no un int
```

Comment:
Hi Diego try commenting on English, please

Reply:
I already changed to english, ty camilo